src/workflow/RunOpenStudioMeasures.cpp

```diff
@@ -19,9 +19,13 @@ void OSWorkflow::runOpenStudioMeasures() {
   // Weather file is handled in runInitialization
 
   LOG(Info, "Beginning to execute OpenStudio Measures");
-  applyMeasures(MeasureType::ModelMeasure, false);
+  applyMeasures(MeasureType::ModelMeasure, ApplyMeasureType::Regular);
   LOG(Info, "Finished applying OpenStudio Measures.");
 
+  LOG(Info, "Beginning to execute Reporting Measures's Model Output Requests");
+  applyMeasures(MeasureType::ReportingMeasure, ApplyMeasureType::ModelOutputRequests);
+  LOG(Info, "Finished applying Reporting Measures's Model Output Requests.")
+
```

Comment:
I made it run right after openstudio modelmeasures. I should probably make a separate job for it for Timings purposes.